packages/webidl/webidl.1.1/opam

```diff
@@ -11,7 +11,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "ocamlfind" {build & >= "1.7.1"}
   "ocamlbuild" {build & >= "0.11.0"}
-  "menhir" {build & >= "20170101"}
+  "menhir" {build & >= "20170101" & < "20211215"}
```

Comment:
Actually same for these ones: http://check.ocamllabs.io/log/1648270349-9d91fc6519d4e3db3bee7883ac155eea5b3b54d3/4.14/good/webidl.1.1